src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,28 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_AGE = 18;
+    private static final int MINIMUM_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getAge() == null || user.getAge() < MINIMUM_AGE) {
+            throw new RuntimeException("Wrong age");
```

Comment:
Instead of using `RuntimeException`, use a custom exception like `RegistrationException`. This will make it clearer that the exception is related to the registration process and not a general runtime error.